manager/integration/tests/backupstore.py

```diff
@@ -213,6 +255,36 @@ def minio_write_backup_cfg_file(client, core_api, volume_name, backup_name, back
         print(err)
 
 
+def backupstore_delete_file(client, core_api, prefix, file_name):
+    backup_target_setting = client.by_id_setting(SETTING_BACKUP_TARGET)
+    backupstore = backup_target_setting.value
+
+    if is_backupTarget_s3(backupstore):
+        return mino_delete_file_in_backupstore(client,
+                                               core_api,
+                                               prefix,
+                                               file_name)
+
+    elif is_backupTarget_nfs(backupstore):
```

Comment:
`else:`
Why not be consistent with `backupstore_create_file`?

Reply:
Fixed